### DISCUSSION TITLE
How to use the trained flux lora?

### DISCUSSION THREAD

**Discussion by bxclib2** [bghira/SimpleTuner#655] [Q&A]
Sorry this maybe a stupid question. But the trained flux lora compatible with comfyUI? How can i use it? Someone can help add a short doc?

Thanks a lot

**Comment by bghira** [comment]
I don't really use webUIs or any kind of UI whatsoever, so I don't really have an answer for you. it should work in ComfyUI, but I wouldn't know how to set it up.

**Reply by bxclib2** [reply]
Oh so the default usage should be the diffuser library am I correct? Thanks

**Reply by kaibioinfo** [reply]
The up-to-date comfyui can load them. The only issue at the moment is that the alpha value is not stored in the lora file. So if you use alpha=rank then you can just use them in Comfy. Otherwise, you have to set the strength of the lora in comfy to alpha/rank.
